ansible/configs/ocp4-cluster/post_software.yml

```diff
@@ -186,7 +186,7 @@
         msg: "{{ item }}"
       loop:
       - "You can access your bastion via SSH:"
-      - "ssh {{ student_name }}@bastion.{{ guid }}{{ subdomain_base_suffix if cloud_provider == 'ec2' else '.'+ocp4_base_domain }}"
+      - "ssh {{ student_name }}@{{ hostvars[bastion_hostname]['ansible_hostname'] }}.{{ guid }}{{ subdomain_base_suffix if cloud_provider == 'ec2' else '.'+ocp4_base_domain }}"
```

Comment:
This is stored as a fact with localhost...can't it be more simple? I think just `{{ bastion_hostname }} should be fine since this task is running on localhost ?

Reply:
Hmm good point, but I haven't tested that. 
Does {{ bastion_hostname }} store only the host part or the full FQDN?

Reply:
Ah, nvm that is gonna grab everything. To be consistent with what we use earlier, we could also use `hostvars[bastion_hostname].shortname`.

Reply:
All-right, will change it into shortname then and have it all consistent.

Reply:
This is where we set it earlier

https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles-infra/infra-common-ssh-config-generate/tasks/main.yml#L42

Reply:
Ok. Updated with the next commit.